odoo/orm/models.py

```diff
@@ -5290,12 +5290,13 @@ def _search(
 
         # security access domain
         if check_access:
+            self_sudo = self.sudo().with_context(active_test=False)
             sec_domain = self.env['ir.rule']._compute_domain(self._name, 'read')
```

Comment:
Is it more pertinent to call `_compute_domain` also with the `with_context(active_test=False)` because we already `optimize(model)` inside. And that's work only because at this level of optimization, we don't use active_test or we do and we have another issue ?